cell_based/test/tutorial/TestCommandLineVertexMeshTutorial.hpp

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
I'm not really sure we need a whole vertex simulation here, it is bloating things a bit for no real benefit (there's no need for it to be a big complicated example, and nothing cell-based specific about passing in arguments). 

What if we just copy and paste the bash script into the `TestCommandLineArgumentsTutorial.hpp` to be rendered as a block of code on the website for readers to paste into a bash script? To e.g. just run the tutorial with different inputs in a loop?

Reply:
I can see this being very useful. Creating multiple tests within `TestCommandLineArgumentsTutorial.hpp` with a commented bash script for each. Then informing the user to copy and paste each variant of the bash script into their file for each test they wish to try out. Each test could then be showing how to execute with various methods, such as using vectors.